test/common/sea.js

```diff
@@ -78,17 +78,18 @@ function injectAndCodeSign(targetExecutable, resource) {
     }
     if (signtoolFound) {
       let certificatesFound = false;
-      let stderr;
+      let stderr, child;
       try {
-        ({ stderr } = spawnSyncAndExitWithoutError('signtool', [ 'sign', '/fd', 'SHA256', targetExecutable ], {}));
+        ({ child, stderr } = spawnSyncAndExitWithoutError('signtool', [ 'sign', '/fd', 'SHA256', targetExecutable ], { status: undefined }));
```

Comment:
This wasn't working because it would have *status === 1* when it fails because of missing certificates. Which means it throws and that leaves *stderr* not set.

So in order to catch this issue, you have to ignore the status and check it here.